forwarder/kafka/kafka_producer.py

```diff
@@ -19,6 +19,8 @@ def _poll_loop(self):
     def close(self):
         self._cancelled = True
         self._poll_thread.join()
+        max_wait_to_publish_producer_queue = 2  # seconds
```

Comment:
Pull out into a constant called something like `PRODUCER_TIMEOUT_SECONDS`?

Reply:
This is specifically a timeout waiting for the producer to publish everything currently in its queue though, not something that we would want to use anywhere else or document anywhere but in the close method I think.

Reply:
I see your point on the locality and, though, I like to avoid them where possible a comment would have be clearer I think.
```
# Give producer opportunity to clear its queue before closing
self._producer.flush(timeout=2)    # seconds
```
But meh, small potatoes ;)